reproc/include/reproc/reproc.h

```diff
@@ -444,6 +444,17 @@ process is completely cleaned up.
 */
 REPROC_EXPORT int reproc_kill(reproc_t *process);
 
+/*!
+Return process ID of child. Return REPROC_EINVAL on error.
```

Comment:
Identifiers are surrounded with backquotes in reproc's code docs so `REPROC_EINVAL` and `reproc_wait` should be surrounded with backquotes in this doc section.